### PR TITLE
Merging documentation branch (testing_docs) rebased on the latest dev branch (marcell-dev)

### PR DESCRIPTION
**Pull request by marcell-ferencz-databricks**

### CODE REVIEW

**Review by robertwhiffin — APPROVED** [review]
LGTM

File: arc/autolinker/autolinker.py [hunk 1]
```diff
@@ -0,0 +1,910 @@
...
+import pyspark
+
+# from dbruntime.display import displayHTML
+
+import splink
+from splink.spark.spark_linker import SparkLinker
+from splink.databricks.enable_splink import enable_splink
+import splink.spark.spark_comparison_library as cl
+
+import hyperopt
```
Comment (milos-colic) [inline arc/autolinker/autolinker.py:13]: @marcell-ferencz-databricks is this listed as part of setup.cfg for arc pacakge?
if yes we need to remove it and mlflow and manually install it as a part of build.
hyperopt and mlflow are provided on platform and we should not package them in the wheel.

Comment (milos-colic) [reply arc/autolinker/autolinker.py:13]: this can be fixed in a separate PR.

**Review by milos-colic — APPROVED** [review]
LGTM for now. We need to solve a couple of things, but we will do this via separate PRs.

**Pull request merged**